tracer/src/Datadog.Trace.ClrProfiler.Managed.Loader/StartupLogger.cs

```diff
@@ -171,5 +171,17 @@ private static bool IsDebugEnabled()
                 return false;
             }
         }
+
+        private static DateTime GetNow()
+        {
+            try
+            {
+                return DateTime.Now;
+            }
+            catch
+            {
+                return DateTime.UtcNow;
+            }
+        }
```

Comment:
Why do we need this change? When would one pass but the other fail? Did you mean to preferentially show `Now` over `UtcNow`? We use UtcNow everywhere else don't we?

Reply:
I wanted to match the same date format as the Managed logs (which uses `.Now`)

Reply:
I removed this method in [a554497](https://github.com/DataDog/dd-trace-dotnet/pull/2533/commits/a55449752300a26ae38d963e7cee70e88212c6ca), and use `DateTime.Now` directly.